config/runtime.exs

```diff
@@ -0,0 +1,23 @@
+import Config
+
+config :cms, CMS.Cache,
+  conn_opts: [
+    host: System.get_env("REDIS_HOST", "127.0.0.1"),
```

Comment:
nit-pick: This should environment variable should be added to the `.envrc.template` file